VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -137,38 +144,47 @@ def device_list_for_billing(self):
         '''
 
         self.disksToBeIncluded = self.disk_list_for_billing()           
-        '''
-            Sample output for lsblk --json command
-            {
-            "blockdevices": [
-            {"name": "sda", "mountpoint": null,
-                "children": [
-                    {"name": "sda1", "mountpoint": null}
-                   ]
-            },
-            {"name": "sdb", "mountpoint": null},
-            {"name": "sdc", "mountpoint": null,
-                "children": [
-                    {"name": "sdc1", "mountpoint": null}
-                   ]
-            },
-            ...
-            ]
-            }
-        '''
         self.logger.log("lsblk o/p {0}".format(self.output_lsblk))
-        if "blockdevices" in self.output_lsblk.keys():
-            for device in self.output_lsblk["blockdevices"]:
-                if "name" in device.keys():
-                    device["name"] = '/dev/' + device["name"]
-                    if device["name"] in self.disksToBeIncluded:
-                        if("children" in device.keys()):
-                                for child in device["children"]:
-                                    if "mountpoint" in child.keys() and child["mountpoint"] != None :
-                                        child["name"] = '/dev/' + child["name"]
-                                        self.devicesToInclude.append(child["name"])
-                                        self.device_mount_points.append(child["mountpoint"])
-            
+        self.logger.log("type of lsblk {0}".format(type(self.output_lsblk)))
+        ''' 
+                NAME          MOUNTPOINT
+                sda
+                sda1          /boot/efi
+                sda2          /boot
+                sda3
+                sda4
+                rootvg-tmplv  /tmp
+                rootvg-usrlv  /usr
+                rootvg-optlv  /opt
+                rootvg-homelv /home
+                rootvg-varlv  /var
+                rootvg-rootlv /
+                sdb
+                sdb1          /mnt/resource
+                sdc
+                sdc1          /mnt/sdc1
+                sdc2          /mnt/sdc2
+                sdc3
+                sde
+                sde1          /mnt/sde1
+                sdf
+                sdg
+        '''
+        for item in self.output_lsblk:
+            item_split = item.split()
+            if(len(item_split)==2):
+                device = item_split[0]
+                mount_point = item_split[1]
+            else:
+                mount_point = ""
```

Comment:
can we log a statement that that item split did not return 2 items and hence skipping?

Reply:
might not need it.